src/app/RoutingRules.php

```diff
@@ -86,6 +87,7 @@ public static function installRoutes($app)
     $app->get('/portal[/{version}[/{topic}[/{path:.*}]]]', PortalPermalinkAction::class);
 
     $app->get('/installation', InstallationAction::class);
+	$app->get('/deprecation', DeprecationAction::class);
```

Comment:
intend

Reply:
good for today. later maybe /features/deprecation or /features/deprecated
but then we can install a redirect.